.github/CONTRIBUTING.rst

```diff
@@ -195,33 +195,23 @@ account. To set this up please go to this
 `page <https://quantumexperience.ng.bluemix.net/qx/account/advanced>`_  and
 register an account.
 
-By default, and if there is no user credentials available, the tests that
-require online access are run with recorded (mocked) information. This is, the
-remote requests are replayed from a ``test/cassettes`` and not real HTTP
-requests is generated. If user credentials are found, in that cases it use them
-to make the network requests.
+By default, and if there is no user credentials available.  If user credentials
```

Comment:
Partial sentence.

Reply:
Oops, I'll update it. I got a bit too greedy with the removals